packages/db/src/repositories/airdrop.ts

```diff
@@ -402,131 +450,18 @@ export class AirdropRepository {
     this.#logger.info("Merkle metadata upserted successfully");
   }
 
-  /**
-   * Get claim status for a specific address
-   */
-  async getClaimStatus(address: string) {
-    try {
-      const normalizedAddress = address.toLowerCase();
-
-      const result = await this.#db
-        .select()
-        .from(claimStatus)
-        .where(eq(claimStatus.address, normalizedAddress))
-        .limit(1);
-
-      if (result.length === 0) {
-        return null;
-      }
-
-      return result[0];
-    } catch (error) {
-      this.#logger.error("Error fetching claim status:", error);
-      throw error;
+  async newSeason(newSeason: NewSeason, tx?: Transaction) {
+    const executor = tx ?? this.#db;
+    const [res] = await executor.insert(seasons).values(newSeason).returning();
+    if (!res) {
+      this.#logger.error("Failed to insert season");
+      throw new Error("Failed to insert season");
     }
-  }
-
-  /**
-   * Get claim statuses for multiple addresses
-   */
-  async getClaimStatusesByAddresses(addresses: string[]) {
```

Comment:
Removed some functions that were copied over from the airdrop repo that we don't use.